### PR TITLE
DOCS-1472: Use case page train and deploy a classification model

### PR DESCRIPTION
**Pull request by sguequierre**
* Steps mostly copy pasted from /ml but edited to focus on classification

### CODE REVIEW

**Review by andf-viam — COMMENTED** [review]
I really like this layout for these Use Cases docs! 

However, I think we need a breif intro and a quick follow-on Next Steps outro to really capitalize on this docs location:
- Needs just a sentence or two before the `table`, introducing the services involved (ML with tooltip, for example) and what we can expect to achieve (success classifying an object in a camera livestream).
- Needs a tiny Next Steps section (or a More Info section). I could envision:
	- Manage and sync your data with the data management service
	- View and filter your images in the Viam app
	- etc

Nice work with this!

File: docs/use-cases/deploy-ml.md [hunk 1]
```diff
@@ -3,6 +3,53 @@ title: "Train and deploy image classification models"
...
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Deploy an image classification model onto your machine with Viam's machine learning (ML) service.
```
Comment (sguequierre) [inline docs/use-cases/deploy-ml.md:14]: @andf-viam Added this intro-- LMK what you think! struggling to add anything for next steps though-- liked your suggestions but data management and view image streams, they've already done, I can't think of what they would want to do post classification walk-through besides maybe export data?

**Review by andf-viam — APPROVED** [review]
LGTM! Suggested a rough Next Steps section, but entirely optional. You're right that all the good stuff is already in the steps themselves. Thank you!

File: docs/use-cases/deploy-ml.md (hunk 1 above)
Comment (sguequierre) [reply docs/use-cases/deploy-ml.md:14]: @andf-viam Thank you so much, this is very helpful! Will adapt from this.

File: docs/use-cases/deploy-ml.md [hunk 2]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+    </td>
+  </tr>
+</table>
+
+## Next steps
+
+After testing your classifier, see the following to further explore Viam's data management and computer vision capabilities:
+
+- [Export Data Using the Viam CLI](/data/export/): Export your synced data from the Viam cloud.
+- [2D Object Detection](/ml/vision/detection/): configure your machine's camera to draw a bounding box around detected objects, based on a machine learning model.
```
Comment (andf-viam) [inline docs/use-cases/deploy-ml.md:62]:
```suggestion
- [2D Object Detection](/ml/vision/detection/): Configure your machine's camera to draw a bounding box around detected objects, based on a machine learning model.
```

Whoops -- my typo in suggestion!

File: docs/use-cases/deploy-ml.md [hunk 3]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Deploy an image classification model onto your machine with Viam's machine learning (ML) service.
```
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:14]: Maybe? I think if it just covers deploying that's not covering the full thing but if it covers more it's not just the service
```suggestion
Create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
```

File: docs/use-cases/deploy-ml.md [hunk 4]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+Do this all on one platform with Viam's data management service: collect data, create a dataset and label it, train or upload an ML model, and deploy the model.
+Then, test if your model works for classifying objects in a camera stream or existing images with the `mlmodel` classification model of vision service.
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
+      <b>1. Collect</b>
+      <p>Start by collecting data from your cameras, sensors, or any other source on your machine with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
```
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:22]: So this is supposed to be specifically for image classification. I don't think sensors or other sources are needed
```suggestion
      <p>Start by collecting data from your cameras with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
```

File: docs/use-cases/deploy-ml.md [hunk 5]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Deploy an image classification model onto your machine with Viam's machine learning (ML) service.
+Do this all on one platform with Viam's data management service: collect data, create a dataset and label it, train or upload an ML model, and deploy the model.
```
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:15]: What all? I think this needs wordsmithing

File: docs/use-cases/deploy-ml.md [hunk 6]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
+      <b>1. Collect</b>
+      <p>Start by collecting data from your cameras, sensors, or any other source on your machine with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/label.svg" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Label data">}}
+      <b>2. Create a Dataset and Label</b>
+      <p>Once you have collected data, <a href="/data/dataset/">label your data and create a dataset</a> in preparation for training machine learning models.</p>
```
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:28]:
```suggestion
      <p>Once you have enough images of the objects you'd like to classify, <a href="/data/dataset/">label your data and create a dataset</a> in preparation for training a classification models.</p>
```

**Review by npentrel — CHANGES_REQUESTED** [review]
I've not finished reviewing but I'd like you to have another go at making this more about image classification before I review further.

File: docs/use-cases/deploy-ml.md (hunk 4 above)
Comment (sguequierre) [reply docs/use-cases/deploy-ml.md:22]: Makes sense, thank you!

File: docs/use-cases/deploy-ml.md [hunk 7]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
+Manage the classification model fully on one platform: collect data, create a dataset and label it, train the model, and deploy it.
+Then, test if your model works for classifying objects in a camera stream or existing images with the `mlmodel` classification model of vision service.
```
Comment (sguequierre) [inline docs/use-cases/deploy-ml.md:16]: @npentrel Still might sound a little awkward but I tried to make more classification-centric

File: docs/use-cases/deploy-ml.md [hunk 8]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
```
Comment (JessamyT) [inline docs/use-cases/deploy-ml.md:14]:
```suggestion
You can create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
```

File: docs/use-cases/deploy-ml.md [hunk 9]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+<table>
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
+      <b>1. Collect</b>
+      <p>Start by collecting images from your cameras with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/label.svg" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Label data">}}
+      <b>2. Create a Dataset and Label</b>
```
Comment (JessamyT) [inline docs/use-cases/deploy-ml.md:27]:
```suggestion
      <b>2. Create a dataset and label</b>
```
[nit] consistency

File: docs/use-cases/deploy-ml.md (hunk 7 above)
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:16]: Yep still a little awkward. You put create and deploy in the  first and second sentence. Try and remove that duplication and see if that fixes it.

**Review by npentrel — CHANGES_REQUESTED** [review]
The lower half of the page hasn't changed I think. Please ensure to also look at that to make it more specific to the use case. Thanks!

File: docs/use-cases/deploy-ml.md (hunk 9 above)
Comment (sguequierre) [reply docs/use-cases/deploy-ml.md:27]: Done!

File: docs/use-cases/deploy-ml.md (hunk 7 above)
Comment (sguequierre) [reply docs/use-cases/deploy-ml.md]: Done

File: docs/use-cases/deploy-ml.md [hunk 10]
```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
...
+  </tr>
+  <tr>
+    <td>
+      <b>4. Deploy your ML model</b>
+      <p>To make use of ML models with your machine, use the built-in <a href="/ml/">ML model service</a> to deploy and run the model.</p>
+    </td>
+  </tr>
+  <tr>
+    <td>{{<imgproc src="/ml/configure.svg" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Configure a service">}}
+      <b>5. Configure a service</b>
```
Comment (npentrel) [inline docs/use-cases/deploy-ml.md:45]:
```suggestion
      <b>5. Configure an mlmodel vision service</b>
```

**Review by npentrel — APPROVED** [review]
One more small change to a header otherwise good to go - thank you, looking great!

File: docs/use-cases/deploy-ml.md (hunk 10 above)
Comment (sguequierre) [reply docs/use-cases/deploy-ml.md:45]: Done

**Pull request merged**